### DISCUSSION TITLE
How to enable build / compiler output to stdout?

### DISCUSSION THREAD

**Discussion by sryze** [microsoft/vcpkg#19590] [Ideas]
Is it possible to have vcpkg show the (verbose) output from the package build script(s) in the console (on Windows) during package installation?

e.g. `vcpkg install package_name --verbose`

I was unable to find an option to achieve this, I'm sorry if this has been asked already. It would be really convenient to have this instead of digging through log files.

**Comment by Be-ing** [comment]
In the meantime look in `vcpkg/buildtrees` for the build logs.

**Comment by JackBoosY** [comment]
Currently no, but maybe we can consider that.
